crates/cli/src/subcommands/publish.rs

```diff
@@ -175,11 +195,14 @@ pub async fn exec(mut config: Config, args: &ArgMatches) -> Result<(), anyhow::E
         let domain = percent_encoding::percent_encode(name_or_identity.as_bytes(), encode_set);
         let mut builder = client.put(format!("{database_host}/v1/database/{domain}"));
 
-        if clear_database != ClearMode::Always {
+        if clear_database == ClearMode::Always {
+            builder = prompt_and_clear(name_or_identity, force, builder)?;
```

Comment:
note that this now only happens in the case where we've passed a `name_or_identity`, but I think this can only clobber something in that case anyway

Reply:
oh actually `--clear-database` _requires_ that to be provided